py3status/modules/gcal.py

```diff
@@ -0,0 +1,170 @@
+# -*- coding: utf-8 -*-
+"""
+Display upcoming calendar events from Google Calendar.
+
+Configuration parameters:
+    agenda_days: show calendar events for number of days (default 1)
+    button_refresh: mouse button to refresh this module (default 2)
+    cache_timeout: refresh interval for this module (default 21600)
+    format: display format for this module (default '{format_event}')
+    format_event: display format for events
+        (default '[\?if=is_date&color=degraded %a %b %d ]
+        [\?if=is_time&color=good %-I:%M %p ][\?color=bad {event}]')
```

Comment:
> color=green

I have a branch that does this but haven't made a PR

Anyhow this does not address the issue that I raised.  Personally I think simple defaults is more important than colorful. at least if you used `color_date` it would make more sense to the user